actionmailbox/db/migrate/20180917164000_create_action_mailbox_tables.rb

```diff
@@ -4,8 +4,11 @@ def change
       t.integer :status, default: 0, null: false
       t.string  :message_id
 
-      t.datetime :created_at, precision: 6, null: false
-      t.datetime :updated_at, precision: 6, null: false
+      if supports_datetime_with_precision?
```

Comment:
I guess the precision is for MySQL 5.6+, since datetime with no precision means unlimited precision for PostgreSQL and SQLite.
If we want explicit precision for the timestamps, I'll create new PR as new migration default in Active Record.

Reply:
Yes, I think the precision is for MySQL. Well, if we could make it omit the precision on Postgres/SQLite and only add it on MySQL if the server supports it, that seems ideal to me.

Reply:
I've just implemented #34970.